ortools/linear_solver/xpress_interface.cc

```diff
@@ -183,6 +205,13 @@ class XpressInterface : public MPSolverInterface {
     return 0.0;
   }
 
+  virtual bool SetSolverSpecificParametersAsString(const std::string& parameters);
+  virtual bool InterruptSolve() {
```

Comment:
You may use the keyword `override` to indicate that you're overriding a `virtual` function of a base class. Then `virtual` becomes implicit.

Reply:
Tagged all overriding functions with `override` in commit 83eb5c65bd5d404684d41bb44da4b5e6941e36d9.